draft-bbs-signatures.md

```diff
@@ -1705,6 +1715,21 @@ Let the prover be in possession of a BBS signature `(A, e, s)` with `A = B * (1/
  <seriesInfo name="Springer," value="Cham"/>
 </reference>
 
+<reference anchor="SGS" target="https://link.springer.com/chapter/10.1007/978-3-540-28628-8_3">
+ <front>
+   <title>Short Group Signatures</title>
+   <author initials="D." surname="Boneh" fullname="Dan Boneh">
+    </author>
+    <author initials="X." surname="Boyen" fullname="Xavier Boyen">
+    </author>
+    <author initials="H." surname="Shacham" fullname="Hovav Scacham">
+    </author>
+    <date year="2004"/>
+ </front>
+ <seriesInfo name="In" value="Advances in Cryptology"/>
+ <seriesInfo name="pages" value="41-55"/>
+</reference>
+
```

Comment:
```suggestion

<reference anchor="ASM06" target="https://link.springer.com/chapter/10.1007/11832072_8">
 <front>
   <title>Constant-Size Dynamic k-TAA</title>
   <author initials="M. H." surname="Au" fullname="Man Ho Au">
    </author>
    <author initials="W." surname="Susilo" fullname="Willy Susilo">
    </author>
    <author initials="Y." surname="Mu" fullname="Yi Mu">
    </author>
    <date year="2006"/>
 </front>
 <seriesInfo name="In" value="International Conference on Security and Cryptography for Networks"/>
 <seriesInfo name="pages" value="111-125"/>
 <seriesInfo name="Springer," value="Berlin, Heidelberg"/>
</reference>
```
I think this should maybe be mentioned too. There is also the [[CL04]](https://link.springer.com/content/pdf/10.1007/978-3-540-28628-8_4.pdf) paper that came before, but it was the above that proved security and "popularized" BBS+ (this is the one that is referenced by [[CDL16]](https://eprint.iacr.org/2016/663.pdf) for example).